app/components/sidebars/main/channels_list/list/index.js

```diff
@@ -61,14 +65,22 @@ function mapStateToProps(state) {
     const canCreatePublicChannels = showCreateOption(state, config, license, currentTeamId, General.OPEN_CHANNEL, isAdmin, isSystemAdmin);
     const canCreatePrivateChannels = showCreateOption(state, config, license, currentTeamId, General.PRIVATE_CHANNEL, isAdmin, isSystemAdmin);
 
+    console.log('--------------');
+    console.log('Unreads', unreadChannelIds);
+    console.log('Favs', favoriteChannelIds);
+    console.log('Channels', orderedChannelIds);
+    console.log('Categories', channelsByCategory);
+
     return {
+        theme: getTheme(state),
+        showLegacySidebar: config.EnableLegacySidebar === 'true',
         canJoinPublicChannels,
         canCreatePrivateChannels,
         canCreatePublicChannels,
-        favoriteChannelIds,
-        theme: getTheme(state),
         unreadChannelIds,
+        favoriteChannelIds,
         orderedChannelIds,
+        channelsByCategory,
```

Comment:
Will have to figure out a clean way of doing https://github.com/mattermost/mattermost-mobile/pull/5460/files#r673952051 here too; maybe something like ` { ...(config.EnableLegacySidebar === true && channelsByCategory) }`.